CMakeSettings.json

```diff
@@ -10,7 +10,13 @@
       "cmakeCommandArgs": "",
       "buildCommandArgs": "-v",
       "ctestCommandArgs": "",
-      "variables": []
+      "variables": [
+        {
+          "name": "UNIT_TESTING",
+          "value": "False",
```

Comment:
I think we should keep this JSON change to avoid VS adding it back every time. I would also be OK with turning Unit testing on by default, if this is only used by IDEs like VS.

Let me know if there is a reason to revert this change.